src/clusterfuzz/_internal/bot/tasks/utasks/__init__.py

```diff
@@ -55,6 +59,14 @@ def _timestamp_now() -> Timestamp:
   return ts
 
 
+def _get_execution_mode(utask_module, job_type):
```

Comment:
Does anything use QUEUE? Why does it exist then?

Reply:
Tasks that run locally use QUEUE. Local tasks call `tworker_preprocess_no_io`,`uworker_main_no_io`, and `tworker_postprocess_no_io`.
For batch/swarming, they use the same path, so we need a way to determine which is used.